app/presenters/product_mailchimp_presenter.rb

```diff
@@ -14,7 +14,7 @@ def json
           id: Digest::MD5.hexdigest(product.id.to_s),
           title: product.name || "",
           description: product.description || "",
-          url: "#{ENV['BASE_URL']}/#{product.category&.permalink || 'products'}/#{product.slug}",
+          url: "#{ENV['BASE_URL']}/#{product.category&.permalink || 'products'}/#{product.slug}" || "",
```

Comment:
@chhga what is this `BASE_URL` ? We should use Rails route helpers to generate an  absolute path